### DISCUSSION TITLE
GitHub Copilot Unavailable After Resubmitting GitHub Student Application

### DISCUSSION THREAD

**Discussion by argf013** [community/community#138017] [GitHub Education] [answered]
### Hi everyone,

Hello guys, I want to ask, why after I resubmitted my GitHub Student application, my GitHub Copilot is no longer available?

**Comment by davevad93** [comment]
Hi @argf013 ,

The free GItHub Copilot subscription comes with the student developer pack, if you have reapplied for your student benefits you'll have to wait for them to become available before you can use Copilot again.

**Reply by argf013** [reply]
How long does it take? Is it the same as the first time I approved?

**Reply by davevad93** [reply, marked as answer]
Yes @argf013 , just like the first time, keep an eye on your application status, when label turns purple and changes to "`Congrats! Your request for @argf013 was approved and your benefits are now available.`" you'll be able to use Copilot again.

**Reply by sneylol** [reply]
Its putple for me but when i login with my github account it says i have to buy a subscribtion.

**Reply by argf013** [reply]
@sneylol i guess u have to claim it first
